### DISCUSSION TITLE
Dark Mode Implementation

### DISCUSSION THREAD

**Discussion by fernandopizarro1** [uprm-inso4101-2023-2024-s2/semester-project-personalfinancemanager#158] [Ideas]
I think a good idea for a feature would be to implement a dark mode, so that the page may be easier to look at, or maybe change the color scheme of the page.

**Comment by diegoq48** [comment]
That sounds good however are you going to make it be a simple toogle that the user can interact with or will it read from the os preferences ie if I have dark mode enabled on windows will it automatically have dark mode on the page or will I have to go out of my way to enable it?

**Reply by fernandopizarro1** [reply]
For now I think the best would be to focus on a toggle that the user can interact with and that the user enables, after we have something implemented then we can start looking towards different implementations.
